lib/templates/deploy.batch.dnx.consoleapp.template

```diff
@@ -0,0 +1,21 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling DNX Console Application deployment.
+
+:: 1. Set WebJob deployment target
+IF NOT DEFINED WEBJOB_DEPLOYMENT_TARGET (
+  SET WEBJOB_DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET%\App_Data\jobs\continuous\deployedJob
+)
+
+:: 2. Create WebJob directory
+IF EXIST "WEBJOB_DEPLOYMENT_TARGET" (
+  CALL :ExecuteCmd mkdir "%WEBJOB_DEPLOYMENT_TARGET%"
+)
+
+:: 3. KuduSync
+CALL :ExecuteCmd "%KUDU_SYNC_CMD%" -v 50 -f "{WEBJOB_PROJECT_PATH}" -t "%WEBJOB_DEPLOYMENT_TARGET%" -n "%NEXT_MANIFEST_PATH%" -p "%PREVIOUS_MANIFEST_PATH%" -i ".git;.hg;.deployment;deploy.cmd"
```

Comment:
The cleaner solution would be to add an optional flag to kudusync to support that. e.g. `--targetSubFolder` would be a relative path that gets combined to the `--to` folder.